spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET posts#index' do
+    it 'returns http success' do
+      get '/users/1/posts/'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'to have template index for all posts' do
+      get '/users/1/posts'
+      expect(response).to render_template(:index)
+    end
+
+    it 'Body incudes correct placeholder text: Posts' do
+      get '/users/1/posts/'
+      expect(response.body).to include('Posts')
+    end
+  end
+
+  describe 'GET posts#index' do

```

Comment:
- [ ] As you have been requested of by the previous reviewer, this should be a test to show and not to index, since index has already been tested above. Kindly make this change.

Reply:
well resolved! ✔️